cmd/validators.go

```diff
@@ -51,9 +53,36 @@ func ValidateFromTo(src, dst string, userSpecifiedFromTo string) (common.FromTo,
 	return userFromTo, nil
 }
 
-const fromToHelpText = "Valid values are two-word phases of the form BlobLocal, LocalBlob etc.  Use the word 'Blob' for Blob Storage, " +
-	"'Local' for the local file system, 'File' for Azure Files, and 'BlobFS' for ADLS Gen2. " +
-	"If you need a combination that is not supported yet, please log an issue on the AzCopy GitHub issues list."
+const fromToHelpFormat = "Valid FromTo are pairs of Source-Destination words (e.g. BlobLocal, BlobBlob) that specify the source and destination resource types. All valid FromTos are: %s"
```

Comment:
We may want to convet that from-to is not required, unless
1. One end is pipe
2. The endpoint does not conform to the normal naming standards.

The issue #2153 is because we parsed source and destination wrong. And even with any valid combo of from-to we would still do that. We could instead make the error clear ( perhaps if we assume user is piping we can throw what the source and destination azcopy saw.)

For case2, I think we should limit to the basic and common options - Blob, File, Local - that fit in for the most customers using a custom domain. These might be confuse the users.

We can instead try improving the Ux  logic for piping and skip modifying from-to help altogether.

Reply:
I think we should still list acceptable values for From-To. I'll indicate that it is optional, and to nudge AzCopy in the right direction.

@normesta thoughts?